src/page/PersonalInformation.page.ts

```diff
@@ -0,0 +1,62 @@
+import { $, ElementFinder, element, by, promise } from 'protractor';
+
+export class PersonalInformationPage {
+
+  public get title(): ElementFinder {
+    return $('.wpb_wrapper > h1');
+  }
+
+  private get finalButton(): ElementFinder {
+    return $('#submit');
+  }
+
+  private async fillFirstNameInput(firstName: string): Promise<void> {
+    return $('input[name="firstname"]').sendKeys(firstName);
+  }
+
+  private async fillLastNameInput(lastName: string): Promise<void> {
+    return $('input[name="lastname"]').sendKeys(lastName);
+  }
+
+  private async fillSexInput(sex: string): Promise<void> {
+    return $(`input[name="sex"][value=${sex}]`).click();
+  }
+
+  private async fillExperienceInput(years: number): Promise<void> {
+    return $(`input[name="exp"][value="${years}"]`).click();
+  }
+
+  private async fillProfessionInput(profession: string): Promise<void> {
+    return $(`input[name="profession"][value="${profession}"]`).click(); 
+  }
+
+  private async fillToolsInput(tool: string): Promise<void> {
+    return $(`input[name="tool"][value="${tool}"]`).click(); 
+  }
+
+  private async fillContinentInput(continent: string): Promise<void> {
+    return $('#continents').element(by.cssContainingText('option', continent)).click();
+  }
+
+  private async fillCommandsInput(command: string): Promise<void> {
+    return $('#selenium_commands').element(by.cssContainingText('option', command)).click();
+  }
+
+  public async fillForm(personalInfo: any): Promise<void> {
```

Comment:
specify a full name for the variables personalInformation is better. The variables must be very clear